src/components/Book.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import './Book.css';
+
+const Book = ({
+  item_id, title, author, category, onRemove,
+}) => (
+  <div className="book-container p-4 border">
+    <div>
+      <ul className="list-unstyled">
+        <li>
+          <div>
+            <div className="book-data d-flex align-items-center justify-content-between ">
+              <div className="book-details">
+                <div className="d-flex flex-column">
+                  <span className="fw-bold gray">{category}</span>
+                  <span className="fw-bold title">{title}</span>
+                  <span className="text-primary">{author}</span>
+                </div>
+                <div className="book-buttons-container mt-3">
+                  <Button variant="light ps-0 text-primary">Comments</Button>
+                  <span className="mx-1">|</span>
+                  <Button variant="light text-primary" onClick={() => onRemove(item_id)}>
+                    Remove
+                  </Button>
+                  <span className="mx-1">|</span>
+                  <Button variant="light text-primary">Edit</Button>
+                </div>
+              </div>
+
```

Comment:
- [ ] Great job staying in accordance with the design. Please remember, we are to follow the design as much as possible. Consider it a test of reproducing a given design. Kindly ensure the add book button is aligned on the right with the book cards like in the original design.
Yours
![Screenshot_5](https://github.com/axcell-0/bookstore/assets/65298048/8c8884a6-7935-437f-a513-fa0e3e78acd5)
original
![Screenshot_1](https://github.com/axcell-0/bookstore/assets/65298048/5c3e5f6b-7bb3-458d-9913-4060da7010de)